src/screens/ContactScreen/ContactScreen.tsx

```diff
@@ -1,32 +1,60 @@
 import React from 'react';
-import { Text } from 'react-native';
-import { SpacerInline } from '@elements';
-import InfoScreen from '../InfoScreen';
+import { Text, View } from 'react-native';
+import { Header, SpacerInline, Title } from '@elements';
+import { Linking } from 'expo';
 import styles from './ContactScreen.styles';
 
 export default () => (
-	<InfoScreen
-		title="Contact."
-		backDestination="ApplicationPendingScreen"
-	>
-		<Text style={styles.documentText}>
-			Please allow 24-48 hours for your registration to complete.  The app will open when your application is processed.
-		</Text>
-		<SpacerInline height={25} />
-		<Text style={styles.headingText}>
-			CALL US
-		</Text>
-		<SpacerInline height={5} />
-		<Text style={styles.documentText}>
-			206-209-0555
-		</Text>
-		<SpacerInline height={25} />
-		<Text style={styles.headingText}>
-			EMAIL US
-		</Text>
-		<SpacerInline height={5} />
-		<Text style={styles.documentText}>
-			admin@thebananaapp.org
-		</Text>
-	</InfoScreen>
+	<View style={styles.outerContainer}>
+		<View style={styles.headerContainer}>
+			<Header showMenu={true} />
+		</View>
+		<Title text="Learn More" />
```

Comment:
Should the title be in a container with padding applied to it rather than using SpacerInlines? That way, the header ('Learn More') is one element resting atop the body element and maintains a composition hierarchy.